server/common/default_config.yaml

```diff
@@ -0,0 +1,55 @@
+
+server:
+  verbose: true
+  debug: false
+  host: "127.0.0.1"
+  port : null
+  scripts : []
+  open_browser: false
+
+multi_dataset:
+  dataroot: null
+
+  # The index page when in multi-dataset mode:
+  #   false or null:  this returns a 404 code
+  #   true:  loads a test index page, which links to the datasets that are available in the dataroot
+  #   string/URL:  redirect to this URL:  flask.redirect(config.multi_dataset__index)
+  index: false
+
+  # A list of allowed matrix types.  If an empty list, then all matrix types are allowed
+  allowed_matrix_types: []
+
+single_dataset:
+  datapath: null
+  obs_names: null
+  var_names: null
+  about: null
+  title: null
+
+annotations:
+  enable: true
+  type: local_file_csv
+  local_file_csv:
+    directory: null
+    file: null
+  ontology:
+    enable: false
+    obo_location: null
+  max_categories: 1000
+
+embeddings:
+  names : []
+  enable_reembedding: false
+
+diffexp:
+  enable: true
+  lfc_cutoff: 0.01
+
+cxg_adaptor:
```

Comment:
would it be useful to create a top-level group `adaptors`, and push the various adaptor config down a level (keep the top namespace clean)?   Ie, `cxg_adaptor` and `anndata_adpator` are children of `adaptor`.   That would also give you a spot to hang generic adaptor config when it arises.

Reply:
Good idea, will do.